ush/sub_gaeac5

```diff
@@ -158,6 +158,7 @@ sbatch=${sbatch:-sbatch}
 ofile=$DATA/subout$$
 >$ofile
 chmod 777 $ofile
+export FI_VERBS_PREFER_XRC=0
```

Comment:
Does this setting resolve what appears to be `mpi_finalize` problems on C5?

Reply:
> Does this setting resolve what appears to be `mpi_finalize` problems on C5?

It appears so.  Here is the notice from Seth Underwood with Gaea C5: "After the C5 update, users reported that some jobs failed during the MPI_Finalize call. We have alerted ORNL and HPE. HPE has suggested setting the environment variable FI_VERBS_PREFER_XRC=0 in the run script (setenv FI_VERBS_PREFER_XRC 0, for csh; export FI_VERBS_PREFER_XRC=0). This has resolved the error in our tests.  Please add this variable to your run script(s) if you also hit this error.  Please note that we do not see any issues preemptively setting this environment variable."  

Now that I think the MPI_Finalize issue is resolved, I am going to adjust the resources and test a little more. I'll let you know when I have my final changes in place for you to look over.